### DISCUSSION TITLE
Active repo?

### DISCUSSION THREAD

**Discussion by matthew73210** [mmcc-xx/BirdCAGE#67] [General]
Hey @mmcc-xx are you still maintaining this repo?

**Comment by simondsmason** [comment]
I think you have your answer.

**Comment by matt-wirth** [comment]
For anyone who sees this later, see here: https://github.com/mmcc-xx/BirdCAGE/discussions/69#discussioncomment-7219204

New repo- https://github.com/matthew73210/BirdCAGE

**Reply by matthew73210** [reply]
> For anyone who sees this later, see here: https://github.com/mmcc-xx/BirdCAGE/discussions/69#discussioncomment-7219204
> 
> 
> 
> New repo- https://github.com/matthew73210/BirdCAGE

No the repo isn't active on my side. I just used it to test the HA integration. The owner never replied and I haven't looked into it since.

**Reply by simondsmason** [reply]
I would like to get this up and running - seems like a really fun project.  I tried building a docker under unraid but was not successful so posted in the unraid forums for help and got nothing.  I would be willing to test but likely need some help getting it going.
